src/Symfony/Component/Scheduler/Trigger/CallbackMessageProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Trigger;
+
+use Symfony\Component\Scheduler\Generator\MessageContext;
+
+final class CallbackMessageProvider implements MessageProviderInterface, \Stringable
+{
+    private \Closure $callback;
+    private string $description;
+
+    public function __construct(callable $callback, string $description = '')
```

Comment:
```suggestion
    public function __construct(callable $callback, private string $description = '')
```
So we can remove the relate line in the constructor.